### PR TITLE
Access Policy that shares event level info with Record Companies

### PR DESCRIPTION
**Pull request by mmwilder26**
Shares album release information with record labels (across RKVST tenancies)

### CODE REVIEW

File: archivist/notebooks/Sharing Album Release Info with Record Labels.ipynb [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+    "                \"arc_display_name\",\n",
+    "                \"arc_display_type\",\n",
+    "                \"arc_description\",\n",
+    "            ],\n",
+    "            \"asset_attributes_write\": [],\n",
+    "            \"behaviours\": ASSET_BEHAVIOURS,\n",
+    "            \"event_arc_display_type_read\": [\"Album Release\"],\n",
+    "            \"event_arc_display_type_write\": [],\n",
+    "            \"include_attributes\": [],\n",
+    "            \"subjects\": [\"subjects/34b291c3-30f4-4d89-9ec7-85f57354f798\"],\n",
```
Comment (JAG-UK) [inline archivist/notebooks/Sharing Album Release Info with Record Labels.ipynb:143]: How is a new user expected to get the subject ID? They will have to have done the Subject exchange dance first. Can we document this or put it in a comment cell?

**Review by JAG-UK — CHANGES_REQUESTED** [review]
One issue - need to make sure people know how to handle Subjects of OBAC policies before they get here.

File: archivist/notebooks/Sharing Album Release Info with Record Labels.ipynb (hunk 1 above)
Comment (mmwilder26) [reply archivist/notebooks/Sharing Album Release Info with Record Labels.ipynb:143]: Hello @JAG-UK  . . . yep I can add some verbiage.  We do have it documented in our "docs",  however those will change but it is there.   I should probably do the same thing for the "other across tenancies" notebook (Sharing Artist Asset with Record Labels")

Comment (mmwilder26) [reply archivist/notebooks/Sharing Album Release Info with Record Labels.ipynb:155]: Hello @JAG-UK   . . . put some comments in the first cell and repeat in the cell with the access policy function.  Didn't want to overwhelm the example with verbiage so I also put a link to documentation.  I can/will update the link as our documentation changes.

**Review by JAG-UK — APPROVED** [review]

**Pull request merged**